src/test/java/com/techsophy/tsf/account/service/UserManagementKeycloakTest.java

```diff
@@ -500,4 +501,39 @@ void assignUserGroupTestTokenEmpty(){
         when(mockTokenUtils.getTokenFromContext()).thenReturn("");
         Assertions.assertThrows(InvalidInputException.class,()->userManagementInKeyCloak.assignUserGroup(userGroupsSchema));
     }
+    @Test
+    void addRolesSucess() throws JsonProcessingException {
```

Comment:
Please add a test case for a wrong client name,i.e. client name not found in keycloak